File: lib/src/main/java/com/hikobe8/logdog/lib/LogDog.java
package com.hikobe8.logdog.lib;

import android.util.Log;

public class LogDog {

    private static final String TAG = "LogDog-----";

    public static int i(String tag, String msg){
        System.out.println(TAG + msg);
        return Log.i(tag, msg);
    }

    public static int d(String tag, String msg){
        System.out.println(TAG +msg);
        return Log.d(tag, msg);
    }

    public static int w(String tag, String msg){
        System.out.println(TAG +msg);
        return Log.w(tag, msg);
    }

    public static int e(String tag, String msg){
        System.out.println(TAG +msg);
        return Log.e(tag, msg);
    }
}

File: lib/src/main/java/com/hikobe8/logdog/lib/LogDog.kt
package com.hikobe8.logdog.lib

import android.util.Log

object LogDog {

    private const val TAG = "LogDog"

    fun i(tag:String, msg:String){
        println(TAG + msg)
        Log.i(tag, msg)
    }

    fun d(tag:String, msg:String){
        println(TAG +msg)
        Log.d(tag, msg)
    }

    fun w(tag:String, msg:String){
        println(TAG +msg)
        Log.w(tag, msg)
    }

    fun e(tag:String, msg:String){
        println(TAG +msg)
        Log.e(tag, msg)
    }

}